lib/breadcrumbs_on_rails/action_controller.rb

```diff
@@ -11,10 +11,14 @@ module BreadcrumbsOnRails
   module ActionController
     extend ActiveSupport::Concern
 
-    included do
+    included do |base|
       extend          ClassMethods
       helper          HelperMethods
       helper_method   :add_breadcrumb, :breadcrumbs
+
+      unless base.respond_to?(:before_action)
```

Comment:
The current master only supports Rails 4+, and `before_action` was [available in Rails 4.0.0](http://api.rubyonrails.org/v4.0.0/classes/AbstractController/Callbacks/ClassMethods.html#method-i-before_action), so I don't think support for `before_filter` is needed.